lib/bootstrap/3.system.coffee

```diff
@@ -1,16 +1,16 @@
 
 require '@nikitajs/filetypes/lib/register'
 
-module.exports = header: "System", handler: ({options}) ->
-  options.locales ?= ['en_US.UTF-8']
-  for username, user of options.users
+module.exports =  hearder: metadata: "System", handler: ({config}) ->
```

Comment:
`hearder: metadata`, really, shouldn't be `metadata: header`, look at nikita unit tests

Reply:
Yes, so true.

And I did it elsewhere. Sorry. I will change this.